.vscode/launch.json

```diff
@@ -26,16 +26,26 @@
       ]
     },
     {
-      "name": "Excel Desktop",
-      "type": "node",
+      "name": "Excel Desktop (Edge Chromium)",
```

Comment:
You don't want to change this launch config since this is using the RN runtime.

When the template is updated to use the shared runtime, then this would be the time to make this launch config change.

Reply:
Well, since this has both custom functions and task pane which can be debugged, you could add "Excel Desktop (Edge Chromium)" for the task pane, but keep the old one renaming "Excel Desktop" to "Custom Functions in Excel Desktop"